x-pack/plugins/index_management/index.js

```diff
@@ -29,7 +29,7 @@ export function indexManagement(kibana)  {
     init: function (server) {
       const router = createRouter(server, PLUGIN.ID, '/api/index_management/');
       server.expose('addIndexManagementDataEnricher', addIndexManagementDataEnricher);
-      registerLicenseChecker(server, PLUGIN.ID);
+      registerLicenseChecker(server, PLUGIN.ID, PLUGIN.NAME, PLUGIN.MINIMUM_LICENSE_REQUIRED);
```

Comment:
Is it necessary to even store `NAME` and `MINIMUM_LICENSE_REQUIRED` in the constants file? I just ask because if not then we can get rid of some indirection by turning those into local variables.